data/json/items/armor/combat_exoskeleton.json

```diff
@@ -0,0 +1,188 @@
+[
+  {
+    "id": "combat_exoskeleton_salvaged",
+    "category": "armor",
+    "type": "ARMOR",
+    "name": { "str": "salvaged combat exoskeleton" },
+    "description": "This exoskeleton has been severely damaged and its electronic parts are completely fried.  You could wear it but the now inactive motors will make moving around very difficult.  Utilizing a variety of webbing, magnets, and clips, specialized armor can be attached to the exoskeleton as one sees fit.",
+    "looks_like": "power_armor_heavy",
+    "symbol": "[",
+    "price": 500000,
+    "price_postapoc": 10000,
+    "volume": "22400 ml",
+    "weight": "8850 g",
+    "flags": [ "STURDY" ],
+    "armor": [
+      {
+        "layers": [ "OUTER" ],
+        "covers": [ "torso", "arm_l", "arm_r", "leg_l", "leg_r" ],
+        "encumbrance": 10,
+        "coverage": 10,
+        "material": [
+          { "type": "steel", "thickness": 5, "covered_by_mat": 50 },
+          { "type": "aluminum", "thickness": 5, "covered_by_mat": 50 },
+          { "type": "kevlar_rigid", "thickness": 5, "covered_by_mat": 50 },
+          { "type": "nylon", "thickness": 1, "covered_by_mat": 100 }
+        ]
+      }
+    ],
+    "pocket_data": [
+      {
+        "pocket_type": "CONTAINER",
+        "volume_encumber_modifier": 0,
+        "description": "Compartment for a UPS.",
+        "flag_restriction": [ "IS_UPS" ],
+        "max_contains_volume": "4000 ml",
+        "moves": 500
+      },
+      {
+        "pocket_type": "CONTAINER",
+        "volume_encumber_modifier": 0,
+        "description": "Compartment for a UPS.",
+        "flag_restriction": [ "IS_UPS" ],
+        "max_contains_volume": "4000 ml",
+        "moves": 500
+      },
```

Comment:
It's the intend to have 2x UPS pockets? For reference, current power armor has just a single one.

Reply:
Yes this is by design. The interview I used as reference talks about the exoskeleton having 2 batteries for x amount of uptime, I understand it as it being able to hold multiple batteries at the same time.

Admittedly the CDDA implementation doesn't **have** to hold multiple UPSes because of it since UPSes function a bit differently than real life batteries.

If being able to hold a second UPS is an issue for whatever reason I'll remove the pocket.